cli/src/test/kotlin/de/deutschepost/sdm/cdlib/archive/ArchiveCommandIntegrationTest.kt

```
@@ -173,8 +174,8 @@ class ArchiveCommandIntegrationTest(@Value("\${artifactory-azure-identity-token}
         val repository = artifactory.repository(repoName)
         try {
             repository.delete(releaseName_build)
-        } catch (_: Exception) {
+        } catch (exception: Exception) {
+            logger.warn("Failed to delete release: ", exception)
```

The original code had an empty catch block, which is a bad practice as it can hide exceptions and make debugging difficult. The fix strategy advised logging the exception. Therefore, I kept the try block intact, and in the catch block, I instantiated the logger if it is not already defined in the surrounding code, using it to log a warning when an exception occurs during the delete operation, following the project's logging conventions and the instructions provided.<br><br>[Related Sonar Issue](http://localhost:9000/issues?open=15156638-4782-4aec-a774-3f852ff7c2ca)<br>[Related Sonar Rule](http://localhost:9000/coding_rules?open=kotlin:S108&rule_key=kotlin:S108)